### PR TITLE
GH-936 Add filter to change alt text in gallery image

### PR DESCRIPTION
**Pull request by Sidsector9**

### CODE REVIEW

File: templates/media/media-gallery-item.php [hunk 1]
```diff
@@ -29,9 +29,15 @@
...
+		<?php
+			global $rtmedia_query;
+
+			$alt_text      = rtmedia_image_alt( false, false );
+			$rtmedia_media = $rtmedia_query->rtmedia;
+		?>
 		<div class="rtmedia-item-thumbnail">
 			<?php echo wp_kses( rtmedia_duration(), array( 'span' => array( 'class' => array() ) ) ); ?>
-			<img src="<?php rtmedia_image( 'rt_media_thumbnail' ); ?>" alt="<?php rtmedia_image_alt(); ?>">
+			<img src="<?php rtmedia_image( 'rt_media_thumbnail' ); ?>" alt="<?php echo apply_filters( 'rtmc_change_alt_text', $alt_text, $rtmedia_media ); ?>">
```
Comment (milindmore22) [inline templates/media/media-gallery-item.php]: add proper escaping function to alt attribute.

File: templates/media/media-gallery-item.php [hunk 2]
```diff
@@ -29,9 +29,15 @@
...
 	   class="<?php echo esc_attr( apply_filters( 'rtmedia_gallery_list_item_a_class', 'rtmedia-list-item-a' ) ); ?>">
 
+		<?php
+			global $rtmedia_query;
+
+			$alt_text      = rtmedia_image_alt( false, false );
+			$rtmedia_media = $rtmedia_query->rtmedia;
+		?>
 		<div class="rtmedia-item-thumbnail">
 			<?php echo wp_kses( rtmedia_duration(), array( 'span' => array( 'class' => array() ) ) ); ?>
```
Comment (milindmore22) [inline templates/media/media-gallery-item.php]: if possible please align array properly.

File: templates/media/media-gallery-item.php (hunk 1 above)
Comment (Sidsector9) [inline templates/media/media-gallery-item.php]: @milindmore22 The changes are done

**Comment by Sidsector9** [comment]
@kiranpotphode please do the final review

File: templates/media/media-gallery-item.php [hunk 3]
```diff
@@ -29,9 +29,24 @@
...
+			$alt_text      = rtmedia_image_alt( false, false );
+			$rtmedia_media = $rtmedia_query->rtmedia;
+		?>
 		<div class="rtmedia-item-thumbnail">
-			<?php echo wp_kses( rtmedia_duration(), array( 'span' => array( 'class' => array() ) ) ); ?>
-			<img src="<?php rtmedia_image( 'rt_media_thumbnail' ); ?>" alt="<?php rtmedia_image_alt(); ?>">
+			<?php
+				echo wp_kses(
+					rtmedia_duration(),
+					array(
```
Comment (kiranpotphode) [inline templates/media/media-gallery-item.php]: @Sidsector9 for more readability you can store array in variable and then pass to function

**Pull request merged**